### PR TITLE
update project 

### PR DESCRIPTION
**Pull request by yigitpolat**

### CODE REVIEW

File: group_vars/aap/aap_config_controller_credential.yml [hunk 1]
```diff
@@ -0,0 +1,53 @@
+---
+controller_credentials:
+  - credential_type: Machine
+    name: "Machine Credential"
+    description: "SSH Machine Connection with IdM credentials"
+    organization: "{{ aap_config_organization }}"
+    inputs:
+      username: "{{ default_ssh_vm_user }}"
+      password: "{{ techuser_ansible_global_password }}"
```
Comment (mmtktl) [inline group_vars/aap/aap_config_controller_credential.yml:9]: Shall we rename this var to more generic name?

File: group_vars/aap/aap_config_controller_job_template.yml [hunk 2]
```diff
@@ -0,0 +1,470 @@
...
+    notification_templates_error: "{{ aap_config_default_notification_template }}"
+    allow_simultaneous: false
+    skip_tags: leapp_remediate
+    survey_enabled: true
+    survey_spec:
+      name: "Survey for remediation"
+      description: "Define remediation points"
+      spec:
+        - question_name: "What do you want to remediate?"
+          question_description: "Inhibitors to remediate, refer to https://lachesis.internal.showroom.run/ui/repo/validated/infra/leapp/content/role/remediate/"
```
Comment (mmtktl) [inline group_vars/aap/aap_config_controller_job_template.yml:216]:
````
```suggestion
          question_description: "Inhibitors to remediate, refer to https://{{ groups.aap_pah | first }}/ui/repo/validated/infra/leapp/content/role/remediate/"
```

File: group_vars/aap/aap_config_controller_job_template.yml [hunk 3]
```diff
@@ -0,0 +1,470 @@
...
+    execution_environment: "{{ aap_config_default_execution_environment }}"
+    credentials: "{{ aap_config_default_credentials }}"
+    allow_simultaneous: false
+    survey_enabled: true
+    survey_spec:
+      name: "Survey for Publish CVs"
+      description: "Define CCVs for CVs"
+      spec:
+        - question_name: "Which CVs within the specified CCV are intended to be published and promoted to the lifecycle environment Dev?"
+          question_description: "Refer to https://aphrodite.internal.showroom.run/content_views"
```
Comment (mmtktl) [inline group_vars/aap/aap_config_controller_job_template.yml:444]:
```suggestion
          question_description: "Refer to https://{{ groups.satellite | first }}/content_views"
```

File: group_vars/aap/aap_config_controller_workflow_template.yml [hunk 4]
```diff
@@ -0,0 +1,375 @@
...
+      spec:
+        - question_name: "RHEL 8.x Hostname or Group?"
+          question_description: "Please provide fully qualified hostname or group name"
+          required: true
+          type: text
+          variable: "host"
+          min: 0
+          max: 250
+        - question_name: "To which lifecycle environment would you like to promote the CCV"
+          question_description: "Refer to https://aphrodite.internal.showroom.run/lifecycle_environments"
```
Comment (mmtktl) [inline group_vars/aap/aap_config_controller_workflow_template.yml:228]:
```suggestion
          question_description: "Refer to https://{{ groups.satellite | first }}/lifecycle_environments"
```

Comment (mmtktl) [inline group_vars/aap/aap_config_controller_workflow_template.yml:228]:
```suggestion
          question_description: "Refer to https://{{ groups.satellite | first }}/lifecycle_environments"
```

File: group_vars/aap/aap_config_controller_workflow_template.yml [hunk 5]
```diff
@@ -0,0 +1,375 @@
...
+      spec:
+        - question_name: "RHEL 9.x Hostname or Group?"
+          question_description: "Please provide fully qualified hostname or group name"
+          required: true
+          type: text
+          variable: "host"
+          min: 0
+          max: 250
+        - question_name: "To which lifecycle environment would you like to promote the CCV"
+          question_description: "Refer to https://aphrodite.internal.showroom.run/lifecycle_environments"
```
Comment (mmtktl) [inline group_vars/aap/aap_config_controller_workflow_template.yml:371]:
```suggestion
          question_description: "Refer to https://{{ groups.satellite | first }}/lifecycle_environments"
```

File: group_vars/aap/aap_config_controller_job_template.yml [hunk 6]
```diff
@@ -0,0 +1,470 @@
...
+    execution_environment: "{{ aap_config_default_execution_environment }}"
+    credentials: "{{ aap_config_default_credentials }}"
+    allow_simultaneous: false
+    survey_enabled: true
+    survey_spec:
+      name: "Survey for Promote CCV"
+      description: "Define CCV"
+      spec:
+        - question_name: "Which CCV you would you like to promote?"
+          question_description: "Refer to https://aphrodite.internal.showroom.run/content_views"
```
Comment (mmtktl) [inline group_vars/aap/aap_config_controller_job_template.yml:412]:
```suggestion
          question_description: "Refer to https://{{ groups.satellite | first }}/content_views"
```

File: group_vars/all/azure_infra.yml [hunk 7]
```diff
@@ -0,0 +1,6 @@
+---
+deployment_environment: dev
+azure_dns_zone: showroom.run
```
Comment (mmtktl) [inline group_vars/all/azure_infra.yml:3]:
```suggestion
azure_dns_zone: example.de
```

File: group_vars/all/git_repo_commit.yml [hunk 8]
```diff
@@ -0,0 +1,11 @@
+---
+git_repo_commit: false
+
+github_organization: redhat-cop
+github_inventory_repo: rhis-inventory
+github_inventory_repo_path: "{{ github_organization }}/{{ github_inventory_repo }}"
+github_code_repo: rhis-code
+github_code_repo_path: "{{ github_organization }}/{{ github_code_repo }}"
+
+git_username: techuser-github-showroom
```
Comment (mmtktl) [inline group_vars/all/git_repo_commit.yml:10]:
```suggestion
git_username: git username
```

File: group_vars/all/git_repo_commit.yml [hunk 9]
```diff
@@ -0,0 +1,11 @@
...
+git_email: bkaraore+showroom@redhat.com
```
Comment (mmtktl) [inline group_vars/all/git_repo_commit.yml:11]:
```suggestion
git_email: git@example.de
```

File: group_vars/all/idm_config.yml [hunk 10]
```diff
@@ -0,0 +1,5 @@
+---
+idm_config: false
+idm_default_dns_zone: "{{ azure_dns_private_dns_zone }}"
+
+ldap_domain_map: "dc=internal,dc=showroom,dc=run"
```
Comment (mmtktl) [inline group_vars/all/idm_config.yml:5]:
```suggestion
ldap_domain_map: "dc={{ azure_dns_private_dnz_zone_subdomain }},dc={{ azure_dns_zone.split('.')[0] }},dc= {{ azure_dns_zone.split('.')[1] }}"
```

File: group_vars/all/imagebuilder.yml [hunk 11]
```diff
@@ -0,0 +1,3 @@
+---
+imagebuilder: true
+rh_activation_key: "ak_showroom_golden_image"
```
Comment (mmtktl) [inline group_vars/all/imagebuilder.yml:3]: should be removed and maybe can be vaulted

File: group_vars/all/pki_idm_generate_certs.yml [hunk 12]
```diff
@@ -0,0 +1,14 @@
+---
+pki_idm_generate_certs: false
+generate_certs: false
+update_ca_trust: false
+
+pki_idm_generate_certs_csr_digest: "aes256"
+pki_idm_generate_certs_csr_common_name: "{{ inventory_hostname }}"
+pki_idm_generate_certs_csr_organization_name: "{{ azure_dns_private_dns_zone | upper }}"
+pki_idm_generate_certs_csr_organization_unit_name: "Showroom Project"
```
Comment (mmtktl) [inline group_vars/all/pki_idm_generate_certs.yml:9]:
```suggestion
pki_idm_generate_certs_csr_organization_unit_name: "RHIS Project"
```

File: group_vars/all/vault_showroom_users.yml
Comment (mmtktl) [inline group_vars/all/vault_showroom_users.yml:1]: File named should be changed

File: group_vars/bastion/public_certificate.yml [hunk 13]
```diff
@@ -0,0 +1,12 @@
+---
+public_certificate: true
+
+public_certificate_common_name: "*.{{ azure_dns_private_dns_zone }}"
+public_certificate_acme_email: showroom-run@outlook.com
```
Comment (mmtktl) [inline group_vars/bastion/public_certificate.yml:5]: should be anonymized.

File: group_vars/github_runner/github_runner.yml [hunk 14]
```diff
@@ -0,0 +1,14 @@
+---
+create_github_runner: true
+
+github_account: "showroom-project"
+runner_org: true
+runner_name: "showroom_runner"
```
Comment (mmtktl) [inline group_vars/github_runner/github_runner.yml:6]:
```suggestion
runner_name: "rhis_runner"
```

File: group_vars/github_runner/github_runner.yml [hunk 15]
```diff
@@ -0,0 +1,14 @@
+---
+create_github_runner: true
+
+github_account: "showroom-project"
```
Comment (mmtktl) [inline group_vars/github_runner/github_runner.yml:4]:
```suggestion
github_account: "rhis-project"
```

File: group_vars/ipahidden/idm_user.yml
Comment (mmtktl) [inline group_vars/ipahidden/idm_user.yml:1]: Users should be anonymous

File: group_vars/rootca/pki_create_rootca.yml [hunk 16]
```diff
@@ -0,0 +1,3 @@
+---
+pki_rootca: true
+ca_common_name: "Showroom RootCA"
```
Comment (mmtktl) [inline group_vars/rootca/pki_create_rootca.yml:3]:
```suggestion
ca_common_name: "RHIS RootCA"
```

File: group_vars/satellite/satellite_config_manifest.yml [hunk 17]
```diff
@@ -0,0 +1,5 @@
+---
+# Variables to upload manifest to Satellite (redhat.satellite.manifest)
+satellite_manifest_path: "~/manifest.zip"
+satellite_manifest_download: true
+satellite_manifest_uuid: "51ee7560-e8b3-441e-b266-f2817db5b39c"
```
Comment (mmtktl) [inline group_vars/satellite/satellite_config_manifest.yml:5]: Should be removed

File: group_vars/satellite/satellite_config_realms.yml [hunk 18]
```diff
@@ -0,0 +1,8 @@
+---
+satellite_config_realms:
+  - name: "INTERNAL.SHOWROOM.RUN"
```
Comment (mmtktl) [inline group_vars/satellite/satellite_config_realms.yml:3]: Should be generated from vars.

Comment (mmtktl) [inline group_vars/satellite/satellite_config_realms.yml:3]: Should be generated from vars.

File: group_vars/satellite/satellite_config_subnets.yml [hunk 19]
```diff
@@ -0,0 +1,61 @@
...
+    from_ip: "10.1.1.4"
+    to_ip: "10.1.1.255"
+    boot_mode: "Static"
+    tftp_proxy: "{{ inventory_hostname }}"
+    dns_proxy: "{{ inventory_hostname }}"
+    template_proxy: "{{ inventory_hostname }}"
+    discovery_capsule: "{{ inventory_hostname }}"
+    ipam: "None"
+    domains:
+      - "internal.showroom.run"
```
Comment (mmtktl) [inline group_vars/satellite/satellite_config_subnets.yml:17]: Should be generated from vars

File: group_vars/satellite/satellite_config_subnets.yml [hunk 20]
```diff
@@ -0,0 +1,61 @@
...
+    from_ip: "10.1.2.4"
+    to_ip: "10.1.2.255"
+    boot_mode: "Static"
+    tftp_proxy: "{{ inventory_hostname }}"
+    dns_proxy: "{{ inventory_hostname }}"
+    template_proxy: "{{ inventory_hostname }}"
+    discovery_capsule: "{{ inventory_hostname }}"
+    ipam: "None"
+    domains:
+      - "internal.showroom.run"
```
Comment (mmtktl) [inline group_vars/satellite/satellite_config_subnets.yml:37]: Should be generated from vars

File: group_vars/satellite/satellite_config_subnets.yml [hunk 21]
```diff
@@ -0,0 +1,61 @@
...
+    from_ip: "10.1.0.4"
+    to_ip: "10.1.0.6"
+    boot_mode: "Static"
+    tftp_proxy: "{{ inventory_hostname }}"
+    dns_proxy: "{{ inventory_hostname }}"
+    template_proxy: "{{ inventory_hostname }}"
+    discovery_capsule: "{{ inventory_hostname }}"
+    ipam: "None"
+    domains:
+      - "internal.showroom.run"
```
Comment (mmtktl) [inline group_vars/satellite/satellite_config_subnets.yml:57]: Should be generated from vars

File: group_vars/satellite/satellite_operation.yml [hunk 22]
```diff
@@ -0,0 +1,67 @@
+---
+satellite_operations: true
+
+satellite_initial_location: "Intranet"
+satellite_initial_organization: "Showroom"
```
Comment (mmtktl) [inline group_vars/satellite/satellite_operation.yml:5]:
```suggestion
satellite_initial_organization: "RHIS"
```

**Review by mmtktl — CHANGES_REQUESTED** [review]

File: group_vars/aap/aap_config_controller_credential.yml (hunk 1 above)
Comment (bkaraoren) [reply group_vars/aap/aap_config_controller_credential.yml:9]: leave it for now

File: group_vars/aap/aap_config.yml [hunk 23]
```diff
@@ -0,0 +1,29 @@
...
+aap_config_default_inventory: "Project SCM Inventory"
+
+aap_config_default_credentials:
+  - "Machine Credential"
+  - "Vault Credential"
+aap_config_default_notification_template: "Slack Notifications"
+aap_config_default_execution_environment: "Automation Hub Default Remote execution environment"
+aap_config_default_job_run: "run"
+
+aap_config_default_scm_branch: devel
```
[the remark this answers is not in the document]
Comment (bkaraoren) [reply group_vars/aap/aap_config.yml]: replaced with main

File: group_vars/aap/aap_config_controller_workflow_template.yml (hunk 4 above)
Comment (bkaraoren) [reply group_vars/aap/aap_config_controller_workflow_template.yml]: replaced

Comment (bkaraoren) [reply group_vars/aap/aap_config_controller_workflow_template.yml]: replaced

File: group_vars/all/azure_infra.yml (hunk 7 above)
Comment (bkaraoren) [reply group_vars/all/azure_infra.yml:3]: we should keep domain for now

File: group_vars/all/git_repo_commit.yml (hunk 9 above)
Comment (bkaraoren) [reply group_vars/all/git_repo_commit.yml]: updated email address, user should be there

File: group_vars/satellite/satellite_operation.yml (hunk 22 above)
Comment (bkaraoren) [reply group_vars/satellite/satellite_operation.yml]: updated

File: group_vars/satellite/satellite_config_subnets.yml (hunk 21 above)
Comment (bkaraoren) [reply group_vars/satellite/satellite_config_subnets.yml]: updated

File: group_vars/satellite/satellite_config_subnets.yml (hunk 20 above)
Comment (bkaraoren) [reply group_vars/satellite/satellite_config_subnets.yml]: updated

File: group_vars/satellite/satellite_config_realms.yml (hunk 18 above)
Comment (bkaraoren) [reply group_vars/satellite/satellite_config_realms.yml]: updated

Comment (bkaraoren) [reply group_vars/satellite/satellite_config_realms.yml]: updated

File: group_vars/satellite/satellite_config_manifest.yml (hunk 17 above)
Comment (bkaraoren) [reply group_vars/satellite/satellite_config_manifest.yml:5]: not sure if we should remove it for now, it is just an example which we used during the project to be able to deploy.

File: group_vars/github_runner/github_runner.yml (hunk 15 above)
Comment (bkaraoren) [reply group_vars/github_runner/github_runner.yml]: updated

File: group_vars/github_runner/github_runner.yml (hunk 14 above)
Comment (bkaraoren) [reply group_vars/github_runner/github_runner.yml:6]: updated

File: group_vars/ipahidden/idm_user.yml
Comment (bkaraoren) [reply group_vars/ipahidden/idm_user.yml:1]: example users we used in the project, can leave as it is.

**Review by bkaraoren — APPROVED** [review]
updated with new project name, LGTM at all to start

**Review by mmtktl — APPROVED** [review]
LGTM

**Pull request merged**